src/apps/_experiments/OswAppWeather.cpp

```diff
@@ -430,19 +483,21 @@ int OswAppWeather::getPrevDay() {
 
 
 void OswAppWeather::setup() {
+OSW_LOG_I("Setup ");
```

Comment:
Could you cleanup these debug-prints? Or remove them from the release-build via `OSW_LOG_D`?